core/input/input.cpp

```diff
@@ -654,7 +654,7 @@ void Input::start_joy_vibration(int p_device, float p_weak_magnitude, float p_st
 	if (p_weak_magnitude < 0.f || p_weak_magnitude > 1.f || p_strong_magnitude < 0.f || p_strong_magnitude > 1.f) {
 		return;
 	}
-	VibrationInfo vibration;
+	VibrationInfo vibration{};
```

Comment:
Maybe this `struct` can be initialized in its declaration instead of when it's used?